yb-voyager/src/queryissue/queryissue.go

```diff
@@ -120,6 +126,31 @@ func (p *ParserIssueDetector) getAllIssues(query string) ([]issue.IssueInstance,
 	return issues, nil
 }
 
+func (p *ParserIssueDetector) GetPercentTypeSyntaxIssues(query string) ([]issue.IssueInstance, error) {
```

Comment:
maybe take the parseTree as an input instead of the query? we already ran parse above.

Reply:
If I did that, I would expose the `pg_query` library in the `queryissue,` which, i think, doesn't make sense as we are using `queryparser`.

Reply:
Hmm got it. Hopefully parsing multiple times is not an perf issue 👍

Reply:
I don't think it will. I remember Sanyam doing a perf test for Parse of pg_query; it was in microseconds.